services/api-server/tests/unit/conftest.py

```diff
@@ -499,7 +499,7 @@ def _patch(href):
             return data.status_href, data.result_href
 
         return mocker.patch(
-            "simcore_service_api_server.services.webserver._get_lrt_urls",
+            "simcore_service_api_server.services_http.webserver._get_lrt_urls",
```

Comment:
TIP: to avoid these it is better if you import the module and then patch the object. It is more resistant to refactoring. Also add `auto_spec=True` to enhance validity of your mock
```python
import simcore_service_api_server.services_http.webserver
mocker.patch.object(simcore_service_api_server.services_http.webserver,._get_lrt_urls", auto_spec=True, side_effect=_get_lrt_urls)
```